### ISSUE TITLE
On start-up asks if I want to download the recent update (Question/Enhancement)

### ISSUE THREAD

**Issue by Scorpion34** [rcmaehl/MSEdgeRedirect#221]
Every time or very frequently I start up my computer MSER starts up to the problem being is I constantly get updates or even when booting it up it asks for an update I was just curious if there's a way to stop the constant updates or even when re-opening the app it asks for an update I do understand that Microsoft consistently changes everything and would still like the updates I'm just tired of  getting asked to update it seems like every 2-3 days now I use Active mode as that was recommended and I expected more of an unobtrusive background app that updates every time system boot up without needing user interface to hit update perhaps a dumb question considering it's in Beta and I have no idea how to code or anything lol but was just curious if something like that is in the future of the app or perhaps a setting that I am just unaware of to change.

Windows 10 Pro
Version: 21H2

**Comment by ElitePheonix2009** [comment]
If you are talking about iefo, than in 0.7.2.0 version you can add scheduled tasks to auto-update it on login / startup.

**Comment by Scorpion34** [comment]
> If you are talking about iefo, than in 0.7.2.0 version you can add scheduled tasks to auto-update it on login / startup.

Yeah it was weird I just did a double take on MSEdgeRedirect and for some reason I was still running 0.7.1.0 despite seeing the 0.7.2.0 update I'm unsure of what you mean by the iefo? Though I use the .exe stable version if that's important to know, because I open the MSER settings and all I see is the version and what I want Microsoft Edge to redirect to.

**Comment by ElitePheonix2009** [comment]
>  I'm unsure of what you mean by the iefo?

I was talking about this:

![image](https://user-images.githubusercontent.com/114980232/201692031-0d9eeb78-6613-4cf9-8024-ae94064de1bb.png)

> Yeah it was weird I just did a double take on MSEdgeRedirect and for some reason I was still running 0.7.1.0 despite seeing the 0.7.2.0 update

You can also download the update from the releases page.

**Comment by MarcGuiselin** [comment]
Basically MSEdgeRedirect keeps a copy of the msedge executable, so you can still open and access edge if you want to.

Periodically this copy needs to be updated, hence the prompt. It needs permission to go into administrator mode to copy the file.

In #223 I propose a change to the way active mode works that would remove the need for the periodic updates.

**Comment by rcmaehl** [comment]
Merging with #223
